src/server/apsystem.test.ts

```diff
@@ -215,6 +215,65 @@ test('ActivityPubSystem - List replies', async t => {
   t.deepEqual(collection.items, [activity.object])
 })
 
+test('ActivityPubSystem - Undo activity', async t => {
+  const store = newStore()
+  const hookSystem = new HookSystem(store, mockFetch)
+  const aps = new ActivityPubSystem('http://localhost', store, mockModCheck, hookSystem, mockServer.log)
```

Comment:
Maybe add a helper function to set up the ActivityPubSystem for these tests. Also, make sure to update the `replies` test.

```javascript
function setupActivityPubSystem () {
  const store = newStore()
  const hookSystem = new HookSystem(store, mockFetch)
  const aps = new ActivityPubSystem('http://localhost', store, mockModCheck, hookSystem, mockServer.log)
  return { store, aps }
}
```

```suggestion
  const { store, aps } = setupActivityPubSystem()
```